nilearn/signal.py

```diff
@@ -31,36 +32,68 @@ def _standardize(signals, detrend=False, normalize=True):
     detrend: bool
         if detrending of timeseries is requested
 
-    normalize: bool
-        if True, shift timeseries to zero mean value and scale
-        to unit energy (sum of squares).
+    standardize: {bool}
+        if True, signal gets standardized
```

Comment:
Could you please documentation details about referring to standardize_strategies though it is not exposed to users.